### PR TITLE
fix GetIndicatorDBotScore IPv6 mapping bug

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/45817

## Description
Fix a bug where IPv6 indicator types are not mapped correctly in GetIndicatorDBotScore script

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
Failed unit-test - https://app.circleci.com/pipelines/github/demisto/content/138993/workflows/4f431349-4bdd-47e7-9e0f-1c340ea43d05/jobs/470763 commit ```f5202be```

File: Packs/CommonScripts/ReleaseNotes/1_6_24.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### GetIndicatorDBotScore
+- Fixed an issue where IPv6 indicator type was not mapped at all to a DBotScoreType.
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/ReleaseNotes/1_6_24.md:4]:
```suggestion
- Fixed an issue where IPv6 indicator types were not mapped to a DBotScoreType.
```

**Review by ShahafBenYakir — APPROVED** [review]
See my comment

File: Packs/CommonScripts/ReleaseNotes/1_6_24.md (hunk 1 above)
Comment (GuyAfik) [reply Packs/CommonScripts/ReleaseNotes/1_6_24.md]: done

**Pull request merged**